pkg/kubectl/cmd/delete.go

```diff
@@ -218,6 +218,8 @@ func (o *DeleteOptions) Validate(cmd *cobra.Command) error {
 			o.WaitForDeletion = true
 			o.GracePeriod = 1
 		}
+	} else if o.ForceDeletion {
+		fmt.Fprintf(o.ErrOut, "warning: --force should be specified with --grace-period=0")
```

Comment:
`warning: --force is ignored because --grace-period is not 0.\n`

To make it clear that `--force` doesn't do anything here and a normal delete operation is sent.